packages/tooltip-react/package.json

```
@@ -0,0 +1,61 @@
+{
+    "name": "@fremtind/jkl-tooltip-react",
+    "version": "0.0.1",
```

Vi gjør vanligvis første versjon av en pakke på denne måten, så blir første publiserte `1.0.0`.

```suggestion
    "version": "1.0.0-alpha.0",
```